docs-web/configuration/transportserver-resource.md

```diff
@@ -156,6 +157,7 @@ The upstream defines a destination for the TransportServer. For example:
 name: secure-app 
 service: secure-app
 port: 8443
+connect-timeout: "30s"
```

Comment:
for consistency with YAML for https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream , could you remove "" from timeout values in the TransportServer doc?